src/d3a/d3a_core/util.py

```diff
@@ -293,11 +292,11 @@ def read_settings_from_file(settings_file):
         advanced_settings = settings["advanced_settings"]
         simulation_settings = {
             "sim_duration": IntervalType('H:M')(
-                settings["basic_settings"].get('sim_duration', timedelta(hours=24))),
+                settings["basic_settings"].get('sim_duration', duration(hours=24))),
             "slot_length": IntervalType('M:S')(
-                settings["basic_settings"].get('slot_length', timedelta(minutes=15))),
+                settings["basic_settings"].get('slot_length', duration(minutes=15))),
             "tick_length": IntervalType('M:S')(
-                settings["basic_settings"].get('tick_length', timedelta(seconds=15))),
+                settings["basic_settings"].get('tick_length', duration(seconds=15))),
```

Comment:
Better to replace default `sim_duration `, `slot_length `, `tick_length ` with some constant_variable.

Reply:
True! Will use the ones from d3a-interface